src/index.js

```diff
@@ -1,17 +1,20 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { BrowserRouter, Route } from 'react-router-dom';
+import CounterContainer from './components/CounterPage/CounterContainer';
+import MainPage from './components/MainPage';
 import './index.css';
```

Comment:
import with styles should be the last and also separated by space line from import above it